src/app/queryrow/queryrow.component.css

```diff
@@ -64,4 +63,13 @@
 method-badge {
     margin:auto;
     height: 22px;
+}
+
+.query-link:hover .ms-Icon--Page:before {
+    content: '\E729';
+}
+
+.query:hover {
+    text-decoration: underline !important;
```

Comment:
Is there a need for the **!important**? I have tried removing it locally and it doesn't break anything.

Reply:
let me confirm.. I had put it coz it was not responding. But I may have changed other things around and it was enabled. Checking now